lib/web_ui/dev/test_runner.dart

```diff
@@ -99,6 +100,9 @@ class TestCommand extends Command<bool> with ArgUtils {
 
   TestTypesRequested testTypesRequested = null;
 
+  /// How many dart2js build tasks are running at the same time.
+  final Pool _pool = Pool(8);
```

Comment:
We should use `BUILD_MAX_WORKERS_PER_TASK` as the pool size. Also, currently we default to 4. I recall you and @mdebbar preferring 4 as the default back when I introduced `BUILD_MAX_WORKERS_PER_TASK`.

Reply:
We should use BUILD_MAX_WORKERS_PER_TASK as the pool size ->> Why?

Also Why @mdebbar prefers 4? 

The highest priority now for Flutter team is to make tree green. One thing we should do is to drop run time for LUCI tasks. I confirmed that all LUCI bots have at least 8 cores. I am ok with an argument for making this higher, I don't see it is 4?

Reply:
> "We should use BUILD_MAX_WORKERS_PER_TASK as the pool size" -> Why? Can you add a link if you made a prior analysis.

Because I am using 32 as the pool size, and it speed up my builds considerably. I added this capability in https://github.com/flutter/engine/pull/17616. Currently, this change removes the capability to configure the pool size.

Reply:
I see, the argument was not every machine has 16 cores. Right now all LUCI machines have at least 8 cores so we can use 8.

Reply:
I believe the argument for 4 cores was in favor of contributors using laptops. A lot of laptops have only 4 cores (although typically they are hyper-threaded, so 8 might still be OK).